### PR TITLE
Rename minio secret and remote

### PR DESCRIPTION
**Pull request by majolo**
To be stacked with PRs on Deploy and Deploy Docs repos.

### CODE REVIEW

File: roles/minio/templates/minio_rclone_secret.yaml.j2 [hunk 1]
```diff
@@ -2,11 +2,14 @@ apiVersion: v1
...
+    seldon-deploy: "true"
 type: Opaque
 stringData:
-  RCLONE_CONFIG_S3_TYPE: s3
-  RCLONE_CONFIG_S3_PROVIDER: minio
-  RCLONE_CONFIG_S3_ENV_AUTH: "false"
-  RCLONE_CONFIG_S3_ACCESS_KEY_ID: "{{ minio_access_key }}"
-  RCLONE_CONFIG_S3_SECRET_ACCESS_KEY: "{{ minio_secret_key }}"
-  RCLONE_CONFIG_S3_ENDPOINT: "http://minio.{{ minio_namespace }}.svc.cluster.local:9000"
+  RCLONE_CONFIG_MINIO_TYPE: s3
```
Comment (RafalSkolasinski) [inline roles/minio/templates/minio_rclone_secret.yaml.j2:10]: Could we make the name of the endpoint customisable with the vars? 
Even if it defaults to MINIO would be nice to have flexibility.

Comment (majolo) [reply roles/minio/templates/minio_rclone_secret.yaml.j2:10]: as in `RCLONE_CONFIG_{minio_remote}_TYPE`?

Comment (RafalSkolasinski) [reply roles/minio/templates/minio_rclone_secret.yaml.j2:10]: If not too verbose `{{ minio_rclone_remote_name }}`

Comment (RafalSkolasinski) [reply roles/minio/templates/minio_rclone_secret.yaml.j2:10]: But yes, something like that would be create. No strong opinion on exact variable name

Comment (majolo) [reply roles/minio/templates/minio_rclone_secret.yaml.j2]: Done 👍🏻

Comment (majolo) [reply roles/minio/templates/minio_rclone_secret.yaml.j2]: Will make sure to test this

**Review by RafalSkolasinski — APPROVED** [review]
Tested, works fine

**Pull request merged**